content/docs/concepts/transformations-chain.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Transformations
+weight: 100
+---
+
+In Fluvio, the data transformation is done using [SmartModules](../../../smartmodules/) - user-defined functions compiled to WASM. A group of SmartModules forms a transformation chain - sequential invocations of each SmartModule according to the order defined upon creation. The output of the first invocation becomes the input of the second, the output of the second becomes the input of the third, and so on.
+
+Both Fluvio Producer and Fluvio Consumer support **Transformation Chaining**. However, the actual code execution happens in different places. For Producer, the transformation
+happens before the data is sent to SPU, inside the Producer's process, hence consuming the resources of the client. For Consumer, the transformation takes place on SPU before the data is sent
```

Comment:
sent to SPU => sent to **the topic on** the SPU

Reply:
consuming the resources => utilizing the resources

(to remove the potential confusion with `consumer` in the next sentence.

Reply:
done